core/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -92,7 +93,7 @@ public void testStoredValue() throws IOException {
         assertTrue(CompressorFactory.isCompressed(new BytesArray(binaryValue2)));
 
         for (byte[] value : Arrays.asList(binaryValue1, binaryValue2)) {
-            ParsedDocument doc = mapper.parse("test", "type", "id", XContentFactory.jsonBuilder().startObject().field("field", value).endObject().bytes());
+            ParsedDocument doc = mapper.parse(SourceToParse.source("test", "type", "id", XContentFactory.jsonBuilder().startObject().field("field", value).endObject().bytes(),XContentType.JSON));
```

Comment:
I'd find some way to split this line. We're *trying* to split lines before 100 characters but that is quite a journey given where we are starting from.